### DISCUSSION TITLE
Annotation creation error in doctrine

### DISCUSSION THREAD

**Discussion by manoveda466** [symfony/symfony#45560] [Q&A]
Hi, I am facing an error while running the command `php bin/console migrations:migrate --no-interaction`.

Doctrine\Common\Annotations\AnnotationException: [Creation Error] The annotation @Authenticated declared on method Zoot\Client\Presenters\EmployeePresenter::actionEmployeeList() does not accept any values, but got {"value":"admin"}.

`	/**`
	 `* @ApiRoute("/api/v1/client/employeess", method="GET", description="Get employeess")`
	 `* @Response(200)`
	 `* @Response(400, code="10001", description="Invalid ordering parameter")`
	`* @Authenticated("admin")`
	` */`
	`public function actionEmployeeList(): void`
	`{`
		`....`
	`}`

**Comment by chyshyya** [comment]
From documentation
```
    /**
     * @IsGranted("ROLE_ADMIN")
     *
     * or use @Security for more flexibility:
     *
     * @Security("is_granted('ROLE_ADMIN') and is_granted('ROLE_FRIENDLY_USER')")
     */
    public function index()
    {
        // ...
    }
```
I think `@Authenticated` is wrong to use and does it exists at all?
